blog/_posts/2022-01-17-scala-3.1.1-released.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: blog-detail
+post-type: blog
+by: Paweł Marks, VirtusLab
+title: Scala 3.1.1 and forward compatibility news
+---
+
+Hello from the Scala 3 team! After a slightly longer than usual pause, we are back with a new release of the Scala compiler and exciting news about improvements in forward compatibility. The delay in the release was caused by urgent bugs that needed backporting and testing, and then the holiday season.
+
+## Improvements is Scala 3.1.1
+
+Scala 3.1.1-RC2 was promoted to a full stable release of the language. Scala 3.1.1 is the first patch version after the 3.1 minor release. It is therefore focused on bug fixes and improvements in overall correctness and stability. Although there are no new additions to the language, you can still spot some noticeable improvements:
+
+- You can use `using` clauses to introduce Scala 2 conversions.
+
+  Now the code like this should compile and perform necessary conversion from `A` to `B` in the method body:
+
+  ```scala
+    def example[A, B](a: A)(using A <:< B) =
+      val b: B = a
+      ???
+  ```
+
+  To know more see [PR #13662](https://github.com/lampepfl/dotty/pull/13662) and [related issue](https://github.com/lampepfl/dotty/issues/12955).
+- Mirrors can be correctly summoned for hierarchical sum types compiled by Scala 3.0.x ([PR #14100](https://github.com/lampepfl/dotty/pull/14100)).
+- Dual Scala 2/3 macros can be now declared in non-experimental scopes ([PR #13795](https://github.com/lampepfl/dotty/pull/13795)).
+- Scaladoc can correctly handle symbolic class names, such as `/` or `\` ([PR #13827](https://github.com/lampepfl/dotty/pull/13827)).
+- Scaladoc documentation has a new, improved UI.
+
+For a comprehensive list of fixes, take a look at the full changelog on GitHub. // TODO: link
+
+## 3.1.2-RC1 and forward compatibility improvements
+
+Scala 3 has excellent backward compatibility guarantees between the minor versions. On the other hand, after the recent release of Scala 3.1, we can see that the libraries should be really cautious with updating the compiler version, as it is forcing the bump on every user of that library. We do not want library authors to be stuck on old versions of the compiler as that would mean that they are locked out of many bugfixes, or we would need to spend enormous effort on backporting every bugfix to all past versioning lines.
```

Comment:
Link to somewhere I can read about this?

Reply:
@prolativ Are you going to publish your documentation of `-Yscala-release`?

Reply:
Waiting for someone's review
https://github.com/scala/docs.scala-lang/pull/2301/files